src/ui-kit/experimental/hierarchical/autocomplete/autocomplete.component.spec.ts

```diff
@@ -364,7 +365,7 @@ export class HierarchicalDataService implements SamHiercarchicalServiceInterface
     return Observable.of(returnItem);
   }
 
-  getHiercarchicalById(id?: string) {
+  getHiercarchicalById(id: string, searchValue: string, sort: Sort, currentItemCount: number): Observable<SamHiercarchicalServiceResult> {
```

Comment:
@thomastighe Decide against an object here?

Reply:
Made that change